packages/dx-grid-core/src/utils/column.js

```diff
@@ -8,3 +8,13 @@ export const getColumnExtension = (columnExtensions, columnName) => {
   }
   return columnExtension;
 };
+
+export const getColumnExtensionValue = (columnExtensions, extensionName, defaultValue) =>
```

Comment:
A function that returns another function? Why do we call it getXXXValue? It feels like it should be named as getXXXValueGetter.

I mean that this looks weird.